### PR TITLE
Update perf test profiles

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-browser/issues/31486
This PR:
* updates perf profiles to the recent onces (zips are stored in GCP, we only store sha1)
* adds a scripts to automate profile updating (but it still should be done by a person);
* do some minor improvements (update chrome_releases.json, fix perf_compare.py, fix handling spaces in win binary path)

## Submitter Checklist:

- [x] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: build/commands/lib/perfTests.js [hunk 1]
```diff
@@ -10,7 +10,10 @@ const path = require('path')
...
 const runPerfTests = (passthroughArgs, perf_config, targets) => {
   targets = targets.replace(/brave:nightly:([^:]+):none:/, "$1:")
-  args = [
+  if (process.platform === 'win32') {
+    targets = '"' + targets + '"'
+  }
+    args = [
     path.join(
       config.braveCoreDir, 'tools', 'perf', 'run_dashboard_perftests.py'),
     '--config=' + perf_config,
```
[the remark this answers is not in the document]
Comment (atuchin-m) [reply build/commands/lib/perfTests.js]: Done

File: build/commands/lib/perfTests.js [hunk 2]
```diff
@@ -10,11 +10,16 @@ const path = require('path')
 
 const runPerfTests = (passthroughArgs, perf_config, targets) => {
   targets = targets.replace(/brave:nightly:([^:]+):none:/, "$1:")
+  if (process.platform === 'win32') {
+    targets = '"' + targets + '"'
```
Comment (goodov) [inline build/commands/lib/perfTests.js:14]: you should just remove this then.

**Review by goodov — APPROVED** [review]

File: tools/perf/update_profile.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      '--component-updater="fast-update"'
+  ]
+
+  # Run browser for 2 minutes to update the profile.
+  # Do this twice to clean the old components.
+  for _ in range(2):
+    logging.info('Run %s', ' '.join(args))
+    with subprocess.Popen(args) as process:
+      logging.info('Waiting for 2 minutes..')
+      time.sleep(120)
```
Comment (iefremov) [inline tools/perf/update_profile.py:68]: i wonder if there is any diagnostic about whether all the components were actually downloaded

**Review by iefremov — APPROVED** [review]

File: tools/perf/update_profile.py (hunk 3 above)
Comment (atuchin-m) [reply tools/perf/update_profile.py:68]: @iefremov I believe we soon change that mechanism to use the recent components.

**Review by bridiver — APPROVED** [review]
build/commands script change is ok

**Pull request merged**